tests/argo-base_test.go

```diff
@@ -135,6 +135,7 @@ data:
     {
     executorImage: $(executorImage),
     containerRuntimeExecutor: $(containerRuntimeExecutor),
+    dockerSockPath: $(dockerSockPath),
```

Comment:
for now, Kubeflow install v2.3.0 argo. does `dockerSockPath` work for v2.3.0?

Reply:
@hougangliu this just adds one config to the workflow-controller-configmap which is not used in v2.3.0. It does not break argo workflow functionality. I have tested this with a fresh install of kubeflow 0.6.2 and run the sample pipeline successfully.